src/Entity/EntityTrait.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Acquia\LiftClient\Entity;
+
+use Symfony\Component\Serializer\Encoder\JsonEncoder;
+use Symfony\Component\Serializer\Normalizer\CustomNormalizer;
+use Symfony\Component\Serializer\Serializer;
+
+trait EntityTrait
+{
+    /**
+   * @param string $key
+   * @param string $default
+   *
+   * @return mixed
+   */
+  private function getEntityValue($key, $default)
```

Comment:
I think we want "protected" here? The argument is SDK is for general-purpose, so devs would extend the entities, and the extended entities need to have access to this function too.